### PR TITLE
Interface/v1.3

### PR DESCRIPTION
**Pull request by Shico-UA**
Total interface changes.

### CODE REVIEW

File: graph_creator/static/graph_creator/custom.js [hunk 1]
```diff
@@ -1,2 +0,0 @@
-var navLi = '#' + $( ".hidden" ).text();
-$(navLi).css('border-bottom', 'solid #65bebb 4px');
```
Comment (DmytryiStriletskyi) [inline graph_creator/static/graph_creator/custom.js]: Fix newline.

File: graph_creator/static/graph_creator/style.css [hunk 2]
```diff
@@ -1,115 +0,0 @@
...
-.table-responsive {
-    border-radius: 10px;
-}
-
-/*Charts*/
-.charts-header {
-    text-align: center;
-    margin-top: 24px;
-    margin-bottom: 0;
-}
```
Comment (DmytryiStriletskyi) [inline graph_creator/static/graph_creator/style.css]: Fix newline.

File: graph_creator/templates/graph_creator/graphs.html [hunk 3]
```diff
@@ -14,46 +14,173 @@
...
+    </div>
+</div>
+</footer>
+</div>
 {% endblock %}
 
 {% block script %}
     <script src="{% static 'graph_creator/graphs.js' %}"></script>
-{% endblock %}
+{% endblock %} }}
```
Comment (DmytryiStriletskyi) [inline graph_creator/templates/graph_creator/graphs.html]: Fix newline.

File: graph_creator/static/graph_creator/custom.js (hunk 1 above)
Comment (Shico-UA) [inline graph_creator/static/graph_creator/custom.js]: Done

File: graph_creator/templates/graph_creator/graphs.html (hunk 3 above)
Comment (Shico-UA) [inline graph_creator/templates/graph_creator/graphs.html]: Done.

**Pull request merged**